tgui/packages/tgui/interfaces/Rockbox.js

```diff
@@ -0,0 +1,87 @@
+
+import { useBackend, useSharedState } from '../backend';
+import { Button, LabeledList, Section, NumberInput, Box, Divider, Tooltip } from '../components';
+import { Window } from '../layouts';
+
+export const Rockbox = (props, context) => {
+  const { act, data }=useBackend(context);
+  const {
+    name,
+    amount,
+    forSale,
+    price,
+    stats,
+  }=data;
+  const [takeAmount, setTakeAmount] = useSharedState(context, 'takeAmount', 1);
+  return (
+
+    <Window
+      title="Rockbox"
+      width={500}
+      height={500}
+      scrollable
+    >
+      <Window.Content>
+        <Section>
+          <Box>{"Take Amount: "}
+            <NumberInput
+              value={takeAmount}
+              width={4}
+              minValue={1}
+              onDrag={(e, value) => setTakeAmount(value)}
```

Comment:
You'll also have to implement `onChange` to let users type into this input if i remember correctly.